### ISSUE TITLE
Uncut images

### ISSUE THREAD

**Issue by ardamamur** [daa233/generative-inpainting-pytorch#44]
can you also provide the original (uncut) images?

**Comment by daa233** [comment]
Sorry, I didn't get the original one before. The cropped images are copied from [deep fill v1](https://github.com/JiahuiYu/generative_inpainting/tree/v1.0.0/examples/imagenet). You may find them in the ImageNet validation set by the image names.

**Comment by ardamamur** [comment]
Thanks
